build.sbt

```diff
@@ -6,14 +6,14 @@ import sbt.Keys.libraryDependencies
 // common settings (apply to all projects)
 organization in ThisBuild := "com.gu"
 version in ThisBuild := "0.2.0"
-scalaVersion in ThisBuild := "2.12.3"
+scalaVersion in ThisBuild := "2.12.8"
```

Comment:
I think Scala 2.12.10 is out now? Arguably we could look at bumping to the 2.13.x series as well, although that'll depend on the libraries we depend on being ready.

Reply:
Have bumped to `2.12.10` might see if we can get to `2.13.x` and update the AWS SDK versions in a separate PR 🚀